### DISCUSSION TITLE
Enable Dashboard use ping ESPHOME

### DISCUSSION THREAD

**Discussion by pbabin2001** [tteck/Proxmox#2966] [Help Needed] [answered]
I'm trying to migrate from ESPHOME addon in Home Assistant to the LXC from your script. The problem is I'm not able to activate the option "Dashboard use ping" as in Home Assistant config. Is anyone already done this? My server run in the 192.168.20.xxx and my iot devise on 192.168.90.xxx.
![esphome](https://github.com/tteck/Proxmox/assets/74836301/209be4aa-901c-432c-b11a-96e4361b2764)

Everything I can find need to add a line in the devise config file but when I validate it, I get an error message.

From Home Assistant addon, everything is fine since I activated the use ping option. This is not a network issue ;)

Thanks for your help.

**Comment by tteck** [comment, marked as answer]
Have you tried adding `Environment=ESPHOME_DASHBOARD_USE_PING=true` to the `esphomeDashboard.service`

e.g.
```
nano /etc/systemd/system/esphomeDashboard.service
```
```
[Unit]
Description=ESPHome Dashboard
After=network.target

[Service]
Environment=ESPHOME_DASHBOARD_USE_PING=true
ExecStart=/usr/local/bin/esphome dashboard /root/config/
Restart=always
User=root

[Install]
WantedBy=multi-user.target
```
This _might_ work.

**Comment by pbabin2001** [comment]
Thanks this is working now!!!

Le sam. 11 mai 2024, à 20 h 17, tteckster ***@***.***> a
écrit :

> Have you tried adding Environment=ESPHOME_DASHBOARD_USE_PING=true to the
> esphomeDashboard.service
>
> e.g.
>
> nano /etc/systemd/system/esphomeDashboard.service
>
> [Unit]
> Description=ESPHome Dashboard
> After=network.target
>
> [Service]
> Environment=ESPHOME_DASHBOARD_USE_PING=true
> ExecStart=/usr/local/bin/esphome dashboard /root/config/
> Restart=always
> User=root
>
> [Install]
> WantedBy=multi-user.target
>
> This *might* work.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tteck/Proxmox/discussions/2966#discussioncomment-9400966>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AR26STIJPMYAVCJESCL4JIDZB2YILAVCNFSM6AAAAABHSJOINSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TIMBQHE3DM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Pascal Babin
(418) 997-5911
***@***.***
